src/main/scala/EmrSparkPlugin.scala

```diff
@@ -49,6 +54,7 @@ object EmrSparkPlugin extends AutoPlugin {
     val sparkS3LoggingFolder = settingKey[Option[String]]("S3 folder for application's logs")
     val sparkS3JsonConfiguration = settingKey[Option[String]]("S3 location for the EMR cluster json configuration")
     val sparkAdditionalApplications = settingKey[Option[Seq[String]]]("Applications other than Spark to be deployed on the EMR cluster, these are case insensitive.")
+    val sparkEMRSteps = settingKey[Option[Seq[StepConfig]]]("Multiple steps to run once cluster is setup")
```

Comment:
I was looking to start a step that will run some scripts on master/worker to set things up before running my Spark app. So I was creating step like you mentioned in build.sbt. Action on failure would be to terminate. Not super convenient. Do you have an alternate?